test/e2e/nft/erc721-interaction.spec.js

```diff
@@ -72,9 +72,12 @@ describe('ERC721 NFTs testdapp interaction', function () {
         assert.equal(await nftsMintStatus.isDisplayed(), true);
 
         // watch 3 of the nfts
-        await driver.clickElement({ text: 'Watch NFT 1', tag: 'button' });
-        await driver.clickElement({ text: 'Watch NFT 2', tag: 'button' });
-        await driver.clickElement({ text: 'Watch NFT 3', tag: 'button' });
+        await driver.fill('#watchNFTInput', '1');
+        await driver.clickElement({ text: 'Watch NFT', tag: 'button' });
+        await driver.fill('#watchNFTInput', '2');
+        await driver.clickElement({ text: 'Watch NFT', tag: 'button' });
+        await driver.fill('#watchNFTInput', '3');
+        await driver.clickElement({ text: 'Watch NFT', tag: 'button' });
 
```

Comment:
we need to update this section as the test dapp layout has changed now, we don't have multiple buttons, but instead just 1 and and input field for setting up the token id

Reply:
oh apologies I think the description was not 100% clear. So the update is already made (see right side), I just wanted explain why I was updating this section of test with the dapp bump :blush: 
Basically the test dapp has changed, and this part needed to be updated to make tests continue passing
does it make sense @legobeat ?

Reply:
thank you v much for reviewing the comments too :heart_eyes: